### DISCUSSION TITLE
Keep Dock Connector

### DISCUSSION THREAD

**Discussion by tomaculum** [dupontgu/retro-ipod-spotify-client#46] [Ideas]
Hey everyone,

some time ago I found this repository: https://github.com/makhowastaken/ipod_usb_c

I used the VCC and ground connections from this repository to power my project and still be able to use my dock connector to charge the device. I cut the original pcb to save space as you can see in the image below. I checked as good as I could to be sure none of those lines cut, did short themselves with the nearby lines. As for now I did not experience any weird behaviour from the remaining pcb - hopefully it stays this way.

![IMG_0963](https://user-images.githubusercontent.com/16617007/120490432-b0a60080-c3b8-11eb-9871-4434bb767f16.jpg)

Maybe it is even possible to use the battery pins and original battery without an extra board?

**Comment by shardsofaperture** [comment]
I like this idea a lot. I purchased a much larger than I should have battery so space is an an extreme premium in my fat 5th gen case even. Thanks for sharing, unfortunately I'm still not up and running with the software yet but this will be the first mod I make when I do!
